app/views/layouts/application.html.erb

```diff
@@ -7,7 +7,6 @@
   <%= stylesheet_link_tag    'application', media: 'all' %>
   <%= javascript_include_tag 'application' %>
   <%= csrf_meta_tags %>
-  <link href='//fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,600,700|Roboto+Slab' rel='stylesheet' type='text/css'>
```

Comment:
@crwallace You're right. LGTM!

https://github.com/department-of-veterans-affairs/va_common/blob/master/app/assets/stylesheets/_va.scss#L8